scripts/python/Ne_from_SLiM_vcfs.py

```diff
@@ -19,49 +19,46 @@ def create_sfs_dict(inpath):
     sfs_dict = {}
 
     # Get files from inpath
-    files = os.listdir(inpath)
+    files = glob.glob(inpath + "**/*.vcf.gz", recursive=True)
 
     for filename in tqdm.tqdm(files):
```

Comment:
Minor change, but it's convention to import the `tqdm` fxn directly (`from tqdm import tqdm`) and then just wrap iterators in `tqdm`